src/containers/about-page.js

```diff
@@ -20,7 +20,6 @@ export default connect(
 )(AboutPage);
 
 function mapStateToProps() {
-  return {};
 }
```

Comment:
Hm. We should probably leave the empty object there. It's less obvious what returning undefined from `mapStateToProps` does.